requirements.txt

```diff
@@ -12,3 +12,5 @@ scikit-learn>=0.22.1
 matplotlib>=3.1.3
 pyhdf>=0.10.1
 seaborn>=0.10.0
+
+hilbertcurve>=2.0.5
```

Comment:
We could put this in a `requirements-test.txt` file that way people who aren't running the tests won't get this installed on their system? There is a `setup.py` option called `tests_require`, but that appears to be deprecated now and I don't know what the best option is...

Reply:
That's really smart. I've added this.